### ISSUE TITLE
Feature reqest: add -m message to allow for a human-authored prefix

### ISSUE THREAD

**Issue by 0atman** [Sett17/turboCommit#7]
I love turbocommit's messages, they're astonishingly detailed. 

I'd love to also in one command add my own words to the start of the message to answer WHY I did something, not just what. gpt (or other devs) will never be able to deduce my motivations. Something like:

```sh
$ turbocommit -m "added foo feature to fix that bug"
```

currently I have to `--amend` my commits to do this.

Thank you!

**Comment by Sett17** [comment]
Do you want this feature to plainly add the typed message to the start of the commit body?
Or did I understand it wrong?

---

Right now you can give it extra information into to prompt, e.g.
```bash
$ turbocommit i did this to make it 3x faster
// or
$ turbocommit 'i did this to make it 3x faster'
```
both of those yield the same prompt.

I had success just adding `use title: added foo feature to fix that bug` and it did adhere to that most of the time (I in general always let it generate at least 2 choices, with the default in the config file)

**Comment by Sett17** [comment]
+ something that gets add to the body of the message would be great for e.g. `fix #6` or other references
